Mueez Afzal-weatherman.py

```diff
@@ -0,0 +1,304 @@
+
+
+class year:
+    def __init__(self):
+          self.name=None
+          self.month_coldest=None
+          self.month_hotest=None
+          self.day_hot=None
+          self.highest=None
+          self.lowest=None
+          self.day_cold=None
+          self.day_humidity=None
+          self.month_humidity=None
+          self.humidity=None
+
+    def get_name(self):
+         return self.name
+    
+    def set_name(self, name):
+          self.name=name
+
+    def check_hot(self,highest):
+        if  self.highest == None or self.highest < highest :
+            return True
+        else:
+            return False
+    
+    def check_cold(self,coldest):
+        if self.lowest == None or self.lowest > coldest:
+            return True
+        else:
+            return False
+
+    def check_humidity(self,humidity):
+        if self.humidity == None or self.humidity < humidity:
+            return True
+        else:
+            return False
+
+     
+    def set_highest(self,month_hotest,day_hot,highest):
+       self.month_hotest=month_hotest  
+       self.day_hot=day_hot
+       self.highest=highest
+    
+    def set_lowest(self,month_coldest,day_cold,lowest):
+       self.month_coldest=month_coldest  
+       self.day_cold=day_cold
+       self.lowest=lowest
+   
+    def set_humidity(self,month_humidity,day_humidity,humidity):
+          self.day_humidity=day_humidity
+          self.month_humidity=month_humidity
+          self.humidity=humidity
+    
+    def print_name(self):
+          print("Year:{} \n Max temperature {} C at {} {} \n Min temperature {} C at {} {} \n Humidity {}% at {} {}".format(self.name,self.highest,self.month_hotest,self.day_hot,self.lowest,self.month_coldest,self.day_cold,self.humidity,self.month_humidity,self.day_humidity))
+
+
+    
+        
+  
+class month:
+    def __init__(self):
+          self.year=None
+          self.name=None
+          self.avg_highest=None
+          self.avg_lowest=None
+          self.humidity=None
+
+    def get_year(self):
+         return self.year
+    def get_month(self):
+         return self.name
+    
+    def set_year(self, name):
+          self.year=name
+
+    def set_data(self,name,avg_lowest,avg_highest,humidity):
+          self.name=name
+          self.avg_highest=avg_highest
+          self.avg_lowest=avg_lowest
+          self.humidity=humidity
+    def print_name(self):
+          print("Year:{} Month:{}  \n Avg Max temperature {} C \n Avg Min temperature {} C \n Avg Mean Humidity {}% ".format(self.year,self.name,self.avg_highest,self.avg_lowest,self.humidity))
+
+class day:
+    def __init__(self):
+          self.year=None
+          self.month=None
+          self.day=None
+          self.highest=None
+          self.lowest=None
+
+    def get_year(self):
+         return self.year
+
+    def get_month(self):
+         return self.month
+ 
+    
+    def set_year(self, name):
+          self.year=name
+
+    def set_hot(self,hot):

```

Comment:
we don't need to write getters and setter in python. we can directly access the object fields using dot operator.